docs/products/postgresql/concepts/pg-shared-buffers.rst

```diff
@@ -0,0 +1,166 @@
+PostgreSQL® shared buffers
+===========================
+
```

Comment:
Add an intro paragraph, e.g.

Learn what shared buffers are and what purposes they have. Find out how using shared buffers impacts performance and how to set them up properly. Finally, discover how to inspect database and query cache performance and put data into cache manually.

Below this paragraph, please add a section title e.g. `About shared buffers` (some subsections inside this one would also be nice).